src/public/Convert-SentinelARArmToYaml.ps1

```diff
@@ -44,63 +75,76 @@ Get-Content -Path "C:\Temp\MyRule.json" -Raw | Convert-SentinelARArmToYaml -OutF
 function Convert-SentinelARArmToYaml {
     [CmdletBinding(DefaultParameterSetName = 'StdOut')]
     param (
-        [Parameter(Mandatory = $true,
+        [Parameter(Mandatory,
             Position = 0,
             ParameterSetName = 'Path')]
-        [Parameter(Mandatory = $true,
+        [Parameter(Mandatory,
             Position = 0,
             ParameterSetName = 'UseOriginalFilename')]
-        [Parameter(Mandatory = $true,
+        [Parameter(Mandatory,
             Position = 0,
             ParameterSetName = 'UseDisplayNameAsFilename')]
-        [Parameter(Mandatory = $true,
+        [Parameter(Mandatory,
             Position = 0,
             ParameterSetName = 'UseIdAsFilename')]
-        [Parameter(Mandatory = $true,
+        [Parameter(Mandatory,
             Position = 0,
             ParameterSetName = 'StdOut')]
         [string]$Filename,
 
         [Alias('Yaml')]
-        [Parameter(Mandatory = $true,
-            ValueFromPipeline = $true,
+        [Parameter(Mandatory,
+            ValueFromPipeline,
             ParameterSetName = 'Pipeline',
             Position = 0)]
         [array]$Data,
 
-        [Parameter(Mandatory = $false,
-            ParameterSetName = 'Path')]
-        [Parameter(Mandatory = $false,
-            ParameterSetName = 'Pipeline')]
+        [Parameter(ParameterSetName = 'Path')]
+        [Parameter(ParameterSetName = 'Pipeline')]
         [string]$OutFile,
 
         [Parameter(ParameterSetName = 'UseOriginalFilename')]
         [switch]$UseOriginalFilename,
 
+        [Parameter(ParameterSetName = 'Pipeline')]
         [Parameter(ParameterSetName = 'UseDisplayNameAsFilename')]
         [switch]$UseDisplayNameAsFilename,
 
+        [Parameter(ParameterSetName = 'Pipeline')]
         [Parameter(ParameterSetName = 'UseIdAsFilename')]
-        [switch]$UseIdAsFilename
+        [switch]$UseIdAsFilename,
+
+        [Parameter(ParameterSetName = 'Pipeline')]
+        [string]$Directory,
```

Comment:
Must use current directory as default value, otherwise the conversion breaks with the following error

![image](https://github.com/f-bader/SentinelARConverter/assets/3834359/d8786617-d606-4438-bb45-c95719a2e479)


```powershell
[string]$Directory = $PWD
```